src/components/Status/Status.props.ts

```diff
@@ -0,0 +1,12 @@
+import { StatusConfig } from './Status.styles';
+import { StatusAppearance } from './StatusAppearance.type';
+import { StatusEmphasis } from './StatusEmphasis.type';
+
+type Status = string;
+
+export type StatusProps = {
+  appearance?: StatusAppearance;
+  custom?: StatusConfig;
+  emphasis?: StatusEmphasis;
+  label?: Status;
```

Comment:
It should be required :)

Reply:
All of the props or just a `label`? @adrian-potepa

Reply:
I'm not sure where (or if) it's defined in designs

Reply:
Label required for sure, the rest have defaults

Reply:
changed